PATTERNS.md

```diff
@@ -131,15 +138,13 @@ The following Shellstyle Patterns would match it:
 ```
 ## EventBridge Patterns
 
-Quamina’s Patterns are inspired by those offered by 
+Quamina’s Patterns are inspired by those offered by
 the AWS EventBridge service, as documented in
 [Amazon EventBridge event patterns](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns.html).
 
-As of release 0.1.1, Quamina supports Exists and
-Anything-But patterns but does not yet support AWS’s 
-`numeric` or `prefix` patterns.  Note that a 
+As of release 1.0, Quamina supports Exists and
```

Comment:
```suggestion
As of release 1.0, Quamina supports `Exists` and
```

Reply:
I think this is consistent. In the doc we describe these as Exists patterns and the syntax to express them is `"exists"`.